src/lib/Controls.scss

```diff
@@ -84,6 +84,21 @@ $controls-button-width: 32px;
         align-items: center;
         justify-content: center;
     }
+
+    .bp-controls-group {
```

Comment:
Do any of the other classes in here need increased specificity? What is being overridden?